openquakeplatform/openquakeplatform/vulnerability/forms.py

```diff
@@ -46,6 +46,7 @@ class Meta:
     intensity_measure_type = forms.ChoiceField(
         choices=INTENSITY_MEASURE_TYPES)
     intensity_measure_unit = ChainedChoiceField(
+        help_text='OPEN ISSUE: if you fail the creation of a new function you must select a different value for "Intensity Measure Type" field and then select again the correct one.',
```

Comment:
I propose (also for the other lines):
`OPEN ISSUE: if the creation of a new function fails, please select a different "Intensity Measure Type" and then re-select the correct one.`
Furthermore, please do not exceed the 80 chars